moveit_ros/planning_interface/moveit_cpp/src/moveit_cpp.cpp

```diff
@@ -67,7 +68,7 @@ MoveItCpp::MoveItCpp(const rclcpp::Node::SharedPtr& node, const Options& options
   if (!loadPlanningSceneMonitor(options.planning_scene_monitor_options))
   {
     const std::string error = "Unable to configure planning scene monitor";
-    RCLCPP_FATAL(LOGGER, error);
+    RCLCPP_FATAL_STREAM(LOGGER, error);
```

Comment:
I don't really understand. What's wrong with not using stream?

Reply:
Rolling doesnt like char * with print style I can keep that style with %s though if you want. Stream solves this without needing any additional formatting